homework/index.js

```diff
@@ -30,13 +30,106 @@
     return elem;
   }
 
+  function reposAbstract(index, data) {
+    const repo = data[index];
+    const leftDiv = document.querySelector('.leftDiv');
+    const table = createAndAppend('table', leftDiv, { class: 'table' });
+    const tBody = createAndAppend('tbody', table);
+    const repositoryRow = createAndAppend('tr', tBody, { class: 'row' });
+    createAndAppend('td', repositoryRow, { text: 'Repository :', class: 'label' });
+    const repositoryRowSecondTd = createAndAppend('td', repositoryRow);
+    createAndAppend('a', repositoryRowSecondTd, {
+      href: repo.html_url,
+      target: '_blank',
+      text: repo.name,
+    });
+    const descriptionRow = createAndAppend('tr', tBody, { class: 'row' });
+    if (repo.description !== null) {
+      createAndAppend('td', descriptionRow, {
+        text: 'Description :',
+        class: 'label',
+      });
+      createAndAppend('td', descriptionRow, { text: repo.description });
+    }
+
+    const forksRow = createAndAppend('tr', tBody, { class: 'row' });
+    createAndAppend('td', forksRow, { text: 'Forks :', class: 'label' });
+    createAndAppend('td', forksRow, { text: repo.forks });
+    const dateRow = createAndAppend('tr', tBody, { class: 'row' });
+    createAndAppend('td', dateRow, {
+      text: 'Updated :',
+      class: 'label',
+    });
+    createAndAppend('td', dateRow, {
+      text: new Date(repo.updated_at).toLocaleString(),
+    });
+  }
+
+  function reposContributor(url) {
+    const root = document.getElementById('root');
+    const rightDiv = document.querySelector('.rightDiv');
+    createAndAppend('p', rightDiv, {
+      class: 'contributor-header',
+      text: 'Contributions',
+    });
+    const ul = createAndAppend('ul', rightDiv, { class: 'contributor-list' });
+    fetchJSON(url, (err, contributors) => {
+      if (err) {
+        createAndAppend('p', root, { text: err.message, class: 'alert' });
+      } else {
+        contributors.forEach(contributor => {
+          const li = createAndAppend('li', ul, {
+            class: 'contributor-item',
+            tabindex: '0',
+            'aria-label': contributor.login,
+          });
+
+          createAndAppend('img', li, {
+            src: contributor.avatar_url,
+            height: 48,
+            class: 'contributor-avatar',
+          });
```

Comment:
The HTML validator produces this error messages for the `img` tag:

```
Error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
```

Reply:
Right, i will do.